### PR TITLE
Added python-munkres-pip to python.yaml

### PR DESCRIPTION
**Pull request by belorenz**
For debian and ubuntu.

Please add the following dependency to the rosdep database.

## Package name: 

python3-munkres3-pip

## Package Upstream Source:

[PyPi munkres3](https://pypi.org/project/munkres3/)

## Purpose of using this:

The Munkres module provides an implementation of the Munkres algorithm (also called the Hungarian algorithm or the Kuhn-Munkres algorithm), useful for solving the Assignment Problem.

Distro packaging links:

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - REQUIRED
- Ubuntu: https://packages.ubuntu.com/
   - REQUIRED
- Fedora: https://src.fedoraproject.org/browse/projects/
  - IF AVAILABLE
- Arch: https://www.archlinux.org/packages/
  - IF AVAILABLE
- Gentoo: https://packages.gentoo.org/
  - IF AVAILABLE
- macOS: https://formulae.brew.sh/
  - IF AVAILABLE
- Alpine: https://pkgs.alpinelinux.org/packages
  - IF AVAILABLE
- NixOS/nixpkgs: https://search.nixos.org/packages
  - OPTIONAL

# Checks
 - [ ] All packages have a declared license in the package.xml
 - [ ] This repository has a LICENSE file
 - [ ] This package is expected to build on the submitted rosdistro

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -2815,6 +2815,22 @@ python-multiprocess-pip:
...
+    packages: [munkres3]
+  debian:
+    pip:
+    packages: [munkres3]
+  osx:
+    pip:
+    packages: [munkres3]
+  ubuntu:
+    pip:
+    packages: [munkres3]
```
Comment (jacobperron) [inline rosdep/python.yaml:2833]:
```suggestion
      packages: [munkres3]
  fedora:
    pip:
      packages: [munkres3]
  debian:
    pip:
      packages: [munkres3]
  osx:
    pip:
      packages: [munkres3]
  ubuntu:
    pip:
      packages: [munkres3]
```

**Review by mabelzhang — COMMENTED** [review]
CI is failing on
```
  ERR: list out of alphabetical order line 2825.  'debian' should come before 'fedora'
  ERR: list out of alphabetical order line 2834.  'python-mysqldb' should come before 'python3-munkres3-pip'
```
Please check the alphabetical ordering indicated in the errors and update. Thanks!

**Comment by hidmic** [comment]
@belorenz friendly ping

**Comment by belorenz** [comment]
> @belorenz friendly ping

Thanks, I lost track of it! Reordered it to alpahbetical order. Fingers are crossed for the checks :)

**Review by hidmic — COMMENTED** [review]
@belorenz hmm, two observations:

- I can see `python3-munkres` available in Debian/Ubuntu and Fedora (see [here](https://pkgs.org/search/?q=python3-munkres)). Why pull it from PyPi?
- In PyPi, [`munkres3`](https://pypi.org/project/munkres3/) looks like a dated predecessor of [`munkres`](https://pypi.org/project/munkres/). The former was last released almost a decade ago, whereas the latter was last released in 2020. Shouldn't be using `munkres` instead?

**Comment by belorenz** [comment]
[Ok, lets goooo!](https://www.youtube.com/watch?v=Jo6fKboqfMs)

**Comment by nuclearsandwich** [comment]
@belorenz if you are still interested in this change could you please re-open the PR and respond to hidmic's comments in https://github.com/ros/rosdistro/pull/32428#pullrequestreview-930232361

I'll close this PR for now.

**Pull request closed without merging**